test/bbl/copy-src.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+if [ ! -d linux ]; then
+  rsync --exclude ".git" -r ../../boards/firechip/firmware/riscv-pk .
```

Comment:
It doesn't matter much, but here's an alternative to `rsync` that ignores uncommitted changes in the working copy:
```sh
git -C ../../boards/firechip/firmware/riscv-pk archive --format=tar --prefix=riscv-pk/ HEAD | tar xf -
```